src/librustdoc/config.rs

```diff
@@ -121,6 +121,8 @@ crate struct Options {
     /// For example, using ignore-foo to ignore running the doctest on any target that
     /// contains "foo" as a substring
     crate enable_per_target_ignores: bool,
+    /// Compile test but do not run them.
```

Comment:
Maybe we could tell others that this is the opposite of `should_test`?

Reply:
This is not the opposite, they do different things (and `no_run` requires `should_test` to also be set).